core/modules/wmgr/python/config.py

```diff
@@ -104,8 +104,8 @@ def dbEngine(self):
         _log.debug('creating new connection %s', kwargs)
         if self.dbPasswd: kwargs['password'] = self.dbPasswd
         inst = getEngineFromArgs(**kwargs)
-        # ignore mysql warnings
-        warnings.filterwarnings("ignore", category=MySQLdb.Warning)
+        # ignore SQLAlchemy warnings
+        warnings.filterwarnings("ignore", category=SAWarning)
```

Comment:
I am not sure that it does what you expect it to do or that it does anything at all. Can you try to remove `filterwarnings()` and see if it integration tests run OK.